service/db/src/main/java/gov/va/vro/service/db/SaveToDbServiceImpl.java

```diff
@@ -30,6 +40,74 @@ public Claim insertClaim(Claim claim) {
     return claim;
   }
 
+  @Override
+  public AssessmentResult insertAssessmentResult(
+      UUID id,
+      String assessmentResult,
+      String veteranIcn,
+      String claimSubmissionId,
+      String idType,
+      String diagnosticCode)
+      throws NoSuchFieldException {
+    ClaimEntity claimEntity =
+        claimRepository
+            .findById(id)
+            .orElseThrow(
+                () ->
+                    new NoSuchFieldException(
```

Comment:
Who exactly will handle this exception in the flow? Since it is inside Camel I am assuming it cannot be the controller. Did we test this?